src/zenml/cli/example.py

```diff
@@ -226,7 +226,7 @@ def latest_release(self) -> str:
         latest_tag = parse(tags[-1].name)
         if type(latest_tag) is not Version:
             return "main"
-        return tags[-1].name  # type: ignore
```

Comment:
I think here it should have stayed `name`.

Reply:
fixed this